Dockerfile.android

```diff
@@ -32,4 +32,4 @@ RUN cp -a /root/files/${mono_version} /root && \
     cd /root && \
     rm -rf /root/${mono_version} /root/godot-mono-builds
 
-CMD /bin/bash
+CMD ["/bin/bash"]
```

Comment:
CMD is supposed to be json formatted

Reply:
That's what it used to be, but containers failed running unless you specified `bash` manually AFAIR.

Reply:
https://github.com/godotengine/build-containers/commit/a5ceea36da322bb7a41e7e714a88c8b46eccb412